client/e2e/src/home.e2e-spec.ts

```diff
@@ -0,0 +1,26 @@
+import { HomePage } from "./home.po";
+import { browser, protractor } from 'protractor';
+
+
+describe('The home page:', () => {
+  let page: HomePage;
+  const EC = protractor.ExpectedConditions;
+
+  beforeEach(() => {
+    page = new HomePage();
+    page.navigateTo();
+  });
+
+  it('Should click the add note button and navigate to the correct page', async () => {
+    let url = await page.getUrl();
+    expect(url.endsWith('/new')).toBe(false);
+
+    await page.clickAddNewNote();
+
+    await browser.wait(EC.urlContains('/new'), 10000);
+
+    url = await page.getUrl();
+    expect(url.endsWith('/new')).toBe(true);
```

Comment:
A little bit redundant—you're already waiting until the browser contains `/new`, so there's no way that this `expect` could *not* be `true`.

But the test will still fail if the url doesn't contain `/new`—it'll just timeout on line 20. That should be fine then ☺️